gcs/gcs.go

```diff
@@ -377,6 +434,97 @@ func FromBytesV1(P uint8, d []byte) (*FilterV1, error) {
 	return &FilterV1{filter: f}, nil
 }
 
+// FilterV2 describes an immutable filter that can be built from a set of data
+// elements, serialized, deserialized, and queried in a thread-safe manner.  The
+// serialized form is compressed as a Golomb Coded Set (GCS) along with the
+// number of members of the set.  The hash function used is SipHash, a keyed
+// function.  The key used in building the filter is required in order to match
+// filter values and is not included in the serialized form.
+//
+// Version 2 filters differ from version 1 filters in three ways:
```

Comment:
They differ in a 4th way too (removal of duplicates), which the PR description calls out.

Reply:
Nice catch.  I forgot to update the comment here when I added the 4th deduplication difference and updated the PR.

Updated.